src/scene/vcFlythrough.cpp

```diff
@@ -373,3 +382,44 @@ void vcFlythrough::UpdateCameraPosition(vcState *pProgramState)
     }
   }
 }
+
+void vcFlythrough::LoadFlightPoints(vcState *pProgramState)
+{
+  int32_t numFrames;
+  udProjectNode_GetMetadataInt(m_pNode, "numFlightPoints", &numFrames, 0);
```

Comment:
I can see that I should remove this variable since the geometry functions have it available